frontend/server/src/Controllers/Contest.php

```diff
@@ -2218,7 +2218,7 @@ public static function columnName(int $idx): string {
     public static function apiClone(\OmegaUp\Request $r): array {
         \OmegaUp\Controllers\Controller::ensureNotInLockdown();
         // Authenticate user
-        $r->ensureMainUserIdentity();
+        $r->ensureIdentityIsOver13();
```

Comment:
```suggestion
        $r->ensureMainUserIdentityIsOver13();
```

there's a bunch of these that should be `ensureMainUserIdentityIsOver13`. Can all of the places where it used to say `ensureMainUserIdentity` that now say `ensureIdentityIsOver13` instead say `ensureMainUserIdentityIsOver13` in this diff?

that is what's causing most of the errors.

Reply:
Ohh yes yes Sure!

Reply:
oops my fault!! I'll correct it.